content/docs/start/data-and-model-versioning.md

```diff
@@ -160,7 +156,7 @@ run it after `git clone` and `git pull`.
 
 <details>
 
-### ⚙️ Expand to explode the project 💣
+### ⚙️ Expand to refresh the project ⟳
```

Comment:
I think they meant "dynamite".
"Refresh" sounds like re-loading the same data to me (wouldn't remove anything).

If we can't find the right phrase, maybe be explicit like "delete cached data".

Reply:
what about "delete locally cached data?"

Reply:
Actually, checking https://dvc.org/doc/start/data-and-model-versioning I don't think we should use the term "cache" here as it's not visible anywhere else, probably intentionally. Maybe keep the original intention for now ("blow up") or go for something like "clear the storage".